desktop/components/main_layout/templates/blank.jade

```diff
@@ -12,8 +12,11 @@ html( data-useragent= userAgent )
       link( type='text/css', rel='stylesheet', href= asset('/assets/#{assetPackage}.css') )
 
   body( class= bodyClass )
-    include ../../modal/template
-    include ../../flash/template
+    unless options && options.modal == false
```

Comment:
Maybe options could have a default? It feels a little funny to have a double negative to check if we _should_ display something. How could we configure defaults in a way that lets us say:
```
if options.modal
 include ../../modal/template
```
Would also generally avoid using a double equals sign, tends to lead to confusing results since it tries to do type conversion.